.stylelintrc.js

```diff
@@ -36,7 +36,7 @@ module.exports = {
         'shorthand-property-no-redundant-values': true,
         'string-no-newline': true,
         'time-min-milliseconds': 100,
-        'unit-allowed-list': ['fr', 'px', 'em', 'rem', '%', 'svh', 'vw', 'vh', 'deg', 'ms', 's', 'dpcm'],
+        'unit-allowed-list': ['fr', 'px', 'em', 'rem', '%', 'svh', 'lvh', 'dvh', 'vw', 'vh', 'deg', 'ms', 's', 'dpcm'],
```

Comment:
It will need QA because you change this file, will svh solve the problem?

Reply:
@farhan-nurzi-deriv it's just linter rule, I've just added two css properties to commit. About `svh`, in my opinion it's edge case and works just when the user has search bar and navigation bars on the mobile screen. Please, check this [link](https://css-tricks.com/the-large-small-and-dynamic-viewports/). So when the user scrolls navigation bar hides and `svh` value is wrong for mobile height. But `dvh` works as our `Div100vhContainer` from component package, it checks resize of the screen and change `dvh` value

Reply:
@farhan-nurzi-deriv As I see you already changed this file 2 month ago and added `svh` to this file)
<img width="1243" alt="image" src="https://github.com/binary-com/deriv-app/assets/120570511/b077a7c6-a357-4574-bde0-64c801ace0f2">

Reply:
Yep2, but my card pass thru QA. Because last time thisyahlen change something in styelint it breaks the styles

Reply:
@farhan-nurzi-deriv it's really strange :) we just added valid values which we can use in the code. Need to ask anyone about this. @yashim-deriv what do you think about this?

Reply:
Design [link](https://www.figma.com/file/6uRANt6IJBb4YjQMQ85Bwa/%5BWallets%5D-Deriv-accounts?type=design&node-id=17776-156871&mode=design&t=qouiB1k1smUXOy1T-0). As I see we don't need to show header in this case

<img width="758" alt="image" src="https://github.com/binary-com/deriv-app/assets/120570511/940b4d91-af1a-41bb-ade0-7ccca5b2122b">

I'll ask our designers in this case